tests/ga/extension_emulator.py

```diff
@@ -32,7 +31,7 @@
 from azurelinuxagent.ga.exthandlers import ExtHandlerInstance
 
 
-class ExtensionCommandName(Enum):
+class ExtensionCommandNames:
```

Comment:
NIT: Dont you need to inherit it from Object (like `ExtensionCommandNames(object)`). 
Not sure if this is needed for py2 vs py3, but we follow this convention for all our classes, it would be cleaner if you could just follow it here too just to be consistent with the other code. Thanks!

Reply:
This is annoying, because SOP for py2 is to inherit directly from `object`, but SOP for py3 is to _not_ directly inherit from `object`. 

Regardless, internal consistency is more important!